AUTHORS.rst

```diff
@@ -22,4 +22,5 @@ Contributors
 * piglei - https://github.com/piglei
 * Anton Gruebel - https://github.com/gruebel
 * Peter Byfield - https://github.com/Peter554
-* Fabian Binz - https://github.com/fbinz
\ No newline at end of file
+* Fabian Binz - https://github.com/fbinz
```

Comment:
Not sure why this has registered as a change 🤔

Reply:
It's just because there is a line ending added to the end of that line.